tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -176,12 +176,13 @@ def _init_devices(self):
                 return
 
             for device in settings.GOLDEN_DEVICES:
+                port, _ = device
```

Comment:
The second element is device type, which should be checked before using OpenThreadController to reset, otherwise, the OpenThreadController will fail.